README.md

```diff
@@ -33,7 +71,7 @@ Run `ng g @nrwl/angular:app my-app` to generate an application.
 
 When using Nx, you can create multiple applications and libraries in the same workspace.
```

Comment:
Looks like missing header here before the NX section to separate it from the authentication.
What do you think about adding the `Developers guide` section, that will unite `generate a library`, `development server`?